db-server/index.js

```diff
@@ -141,7 +142,6 @@ function createServer(db) {
   api.del('/sessionToken/:id', withIdAndBody(db.deleteSessionToken))
   api.put('/sessionToken/:id', withIdAndBody(db.createSessionToken))
   api.post('/sessionToken/:id/update', withIdAndBody(db.updateSessionToken))
-  api.get('/sessionToken/:id/device', withIdAndBody(db.sessionToken))
```

Comment:
I'm pretty sure we intended to remove this API but never got around to it; ref https://github.com/mozilla/fxa-auth-server/pull/2934